.travis.yml

```diff
@@ -2,13 +2,19 @@ language: ruby
 sudo: false
 before_install:
   - which bundle || gem install bundler
+script:
+  - bundle list
+  - bundle exec rake
 rvm:
   - 2.2.7
   - 2.3.4
   - 2.4.1
   - ruby-head
   - jruby-9.1.8.0
   - jruby-head
+gemfile:
+  - Gemfile
+  - gemfiles/Gemfile.rack-1
```

Comment:
@perlun ok, it looks good. As we still have only 2 kind of `Gemfile`, moving it to the top-level makes sense. Let me try it.